app/activity/[id]/graphql/query/activityCollection.graphql

```diff
@@ -0,0 +1,25 @@
+query activityCollection($uuid: UUID!) {
+  activityCollection(filter: { uuid: { eq: $uuid } }) {
+    edges {
+      node {
+        id
+        uuid
+        trip_id
+        title
+        time_from
+        time_to
+        address
+        url
+        memo
+        cost
+        image_storage_object_id
+        trips {
+          id
+          uuid
+          user_id
+          title
+        }
```

Comment:
I have a question! When will you use trips in activity?

Reply:
Actually, we don't need trip info. I just got that just in case. I removed it !